cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -62,6 +60,11 @@
     @Parameter(names = {Param.SOURCE_REGEX_LONG, Param.SOURCE_REGEX_SHORT}, arity = 1, required = false, description = Param.SOURCE_REGEX_DESCRIPTION)
     String sourcePathFilterRegex;
 
+    @Parameter(names = {"--inheritance-mode"}, required = false, description = "How to handle missing translations "
+            + "the parent (USE_PARENT for using parent). ",
+            converter = LocalizedAssetBodyInheritanceMode.class)
+    LocalizedAssetBody.InheritanceMode inheritanceMode = null;
```

Comment:
Initialize with default value: USE_PARENT. That way the documentation shows the default value